lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -1,271 +1,273 @@
-                 [class.adf-datatable__header--sorted-asc]="isColumnSorted(col, 'asc')"
-                 [class.adf-datatable__header--sorted-desc]="isColumnSorted(col, 'desc')"
-                 (click)="onColumnHeaderClick(col)"
-                 (keyup.enter)="onColumnHeaderClick(col)"
-                 role="columnheader"
-                 [attr.tabindex]="isHeaderVisible() ? 0 : null"
-                 [attr.aria-sort]="col.sortable ? (getAriaSort(col) | translate) : null"
-                 adf-drop-zone dropTarget="header" [dropColumn]="col">
-                <ng-container *ngIf="!col.header">
-                    <span *ngIf="col.title" class="adf-datatable-cell-value">{{ col.title | translate}}</span>
-                    <span *ngIf="col.title && col.sortable" class="adf-sr-only" aria-live="polite">{{ getSortLiveAnnouncement(col) | translate: { string: col.title | translate } }}</span>
-                    <ng-template *ngIf="allowFiltering" [ngTemplateOutlet]="headerFilterTemplate" [ngTemplateOutletContext]="{$implicit: col}"></ng-template>
-                </ng-container>
-                <ng-template *ngIf="col.header" [ngTemplateOutlet]="col.header" [ngTemplateOutletContext]="{$implicit: col}"></ng-template>
-            </div>
-            <!-- Actions (right) -->
-            <div *ngIf="actions && actionsPosition === 'right'" class="adf-actions-column adf-datatable-cell-header adf-datatable__actions-cell">
-                <span class="adf-sr-only">{{ 'ADF-DATATABLE.ACCESSIBILITY.ACTIONS' | translate }}</span>
-            </div>
-        </adf-datatable-row>
-        <mat-form-field *ngIf="display === 'gallery' && isHeaderVisible()">
-            <mat-select [value]="getSortingKey()" [attr.data-automation-id]="'grid-view-sorting'">
-                <mat-option *ngFor="let col of getSortableColumns()"
-                            [value]="col.key"
-                            [attr.data-automation-id]="'grid-view-sorting-'+col.title"
-                            (click)="onColumnHeaderClick(col)"
-                            (keyup.enter)="onColumnHeaderClick(col)">
-                    {{ col.title | translate}}
-                </mat-option>
-            </mat-select>
-        </mat-form-field>
-    </div>
-
-    <div class="adf-datatable-body" role="rowgroup">
-        <ng-container *ngIf="!loading && !noPermission">
-            <adf-datatable-row *ngFor="let row of data.getRows(); let idx = index"
-                [row]="row"
-                (select)="onEnterKeyPressed(row, $event)"
-                (keyup)="onRowKeyUp(row, $event)"
-                (keydown)="onRowEnterKeyDown(row, $event)"
-                [adf-upload]="rowAllowsDrop(row)"
-                [adf-upload-data]="row"
-                [ngStyle]="rowStyle"
-                [ngClass]="getRowStyle(row)"
-                [attr.data-automation-id]="'datatable-row-' + idx">
+<div class="adf-datatable-container">
```

Comment:
is this the only change we have done ? why is needed and before wasn't ?

Reply:
before, we did not have the scroll bar horizontally, it looks like in the image attached to the issue https://alfresco.atlassian.net/browse/ADF-5466

Reply:
Still not clear, on regression env we did not have the adf-datatable-container, and the resize was working, right?
If the previous is true, my question is why do we need it now?

Reply:
yes, the resize was working because we had scroll only for datatable-body( but not included header row), it was just scrolling the body part as shown below, so as in the issue ticket, it's mentioned as the columns are not aligned. Now the whole datatable has the scroll including the header row
![image](https://user-images.githubusercontent.com/58967843/141464880-5e415827-06ad-4be9-9e0d-736c42ad8364.png)